packages/react-scripts/template/package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
OK, but I think it has no value on it's own (as a PR), it's just theoretically wrong to say that the package.json path is react-scripts/package.json, it should be the template/package.json. Right now it's not an issue but maybe in the future